### DISCUSSION TITLE
Unable to accept array in request

### DISCUSSION THREAD

**Discussion by robertchar92** [goravel/goravel#412] [General]
type AdminCreateRequest struct {
	Name           string   `form:"name" json:"name"`
	Code           string   `form:"code" json:"code"`
	ProductID   []uint64 `form:"product_id[]" json:"product_id[]"`
}

got error 
json: cannot unmarshal string into Go struct field AdminCreateRequest.product_id[] of type []uint64

**Comment by robertchar92** [comment]
i think it because goravel accept all request in string format, because i have other request where i try to accept id as int it got error to and i need to add string behind field for it to work. 

ProductID uint64 `form:"product_id,string" json:"product_id,string"` 

i need to do it like this for it to work

**Reply by hwbrzzl** [reply]
Added an example for this case: https://github.com/goravel/example/pull/15
